comms/DiskModule.cpp

```diff
@@ -416,7 +417,7 @@ class CDiskModule : public IDiskModule, public clib::CLockableUnknown
     bool IsPlugin() const
     {
         clib::recursive_mutex::scoped_lock proc(m_mutex);
-        return m_plugin || m_parent == NULL;
```

Comment:
I think (from memory) that "IsPlugin" might also be used for coloring the root folders and for some special handling for "root" folders.  Can you track down all the places it is currently being called from and double check?

If so, we may want two methods:  "IsPlugin" and "IsRoot"